server/tests.json

```diff
@@ -0,0 +1,17 @@
+{
+	"config": {
+		"test": [ "parser" ]
+	},
+	"devdocs": {
+		"test": [ "index" ]
+	},
+	"i18n": {
+		"test": [ "test" ]
+	},
+	"i18nlint": {
```

Comment:
We are removing `i18nlint` folder with https://github.com/Automattic/wp-calypso/pull/3849.